specification/cognitiveservices/data-plane/Language/preview/2022-05-15-preview/examples/SuccessfulAnalyzeTextJobsCancelRequest.json

```diff
@@ -8,7 +8,7 @@
   "responses": {
     "202": {
       "headers": {
-        "Operation-Location": "{Endpoint}/language/analyze-text/jobs/{jobId}?api-version=2022-05-01-preview"
+        "Operation-Location": "{Endpoint}/language/analyze-text/jobs/{jobId}?api-version=2022-05-15-preview"
```

Comment:
More of service feedback, but our new LRO guidelines (internal doc currently) prescribe not sending an `api-version` back in LRO or pageable links. I don't think it's wrong per-se and this wouldn't hold up the feature, but more of an FYI.

Reply:
Keeping it as is for now /